packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -189,7 +192,20 @@ export class DebugConfigurationManager {
         }
     }
     async addConfiguration(): Promise<void> {
-        const { model } = this;
+        let model: DebugConfigurationModel | undefined;
+        if (this.models.size > 1) {
+            model = await this.quickPick.show(Array.from(this.models.entries()).map(entry => {
+                const [rootFolder, modelValue] = entry;
+                return {
+                    label: this.labelProvider.getName(new URI(rootFolder)),
+                    value: modelValue
+                };
+            }), {
+                placeholder: 'Select for which root folder to add the configuration to'
```

Comment:
I think "workspace folder" would be the VS Code terminology here: I suggest:

> Select the workspace folder to add the configuration to

Reply:
done